backend/app/api/src/endpoints/organization/dashboard/registration-periods/PatchOrganizationRegistrationPeriodsEndpoint.ts

```diff
@@ -398,6 +398,11 @@ export class PatchOrganizationRegistrationPeriodsEndpoint extends Endpoint<Param
             model.defaultAgeGroupId = await this.validateDefaultGroupId(struct.defaultAgeGroupId);
         }
 
+        // if the group is moved to another period
+        if (struct.periodId !== undefined && !(period && period.id === struct.periodId)) {
+            period = await RegistrationPeriod.getByID(struct.periodId);
```

Comment:
Hier moet nog validatie komen denk ik? Misschien niet hier, maar op een andere plaats als de werking ook wat wordt gewijzigd

Reply:
Ook eventueel kijken of je wel toegangsrechten hebt

Reply:
(je moet wel nog steeds kijken of het gewijzigd is, want je moet dan registrations periodId ook wijzigen)

Reply:
het is niet omdat je toegang hebt tot de group dat je toegang hebt tot een willekeurig andere period - inderdaad best enkel voor hoofdbeheerders

Reply:
+ niet mogelijk indien period locked is